src/State/taskActions.ts

```diff
@@ -0,0 +1,46 @@
+export const ADD_TASK = "ADD_TASK";
```

Comment:
Do we still need these actions? because below you are using `ADD_TASK_STARTED`

Reply:
Yes because Add_Task and all others are being passed in sagas. Also the Task Action Files contains these actions as Types